lib/msf/core/post/windows/version.rb

```diff
@@ -0,0 +1,150 @@
+# -*- coding: binary -*-
+
+module Msf::Post::Windows::Version
+include Msf::Post::Windows::Registry
```

Comment:
Small indentation issue:
```suggestion
  include Msf::Post::Windows::Registry
```